src/main/webapp/app/shared/metis/metis.service.ts

```diff
@@ -368,6 +369,39 @@ export class MetisService implements OnDestroy {
         return `/courses/${this.getCourse().id}/lectures/${lectureId}`;
     }
 
+    /**
+     * returns the router link required for navigating to the exam
+     * @param {string} lectureId ID of the exam to be navigated to
```

Comment:
```suggestion
     * @param {string} exam ID of the exam to be navigated to
```
I think this should be exam ID

Reply:
Yes, you are right :D